File: problemas/ad-hoc/1217/1217.py
pesoPorDia = 0.0
gastoPorDia = 0.0

N = int(input())

for i in range(1, N + 1):
    V = float(input())
    frutas = len(input().strip().split(' '))

    print(f"day {i}: {frutas} kg")

    pesoPorDia += frutas
    gastoPorDia += V

pesoPorDia /= N
gastoPorDia /= N

print(f"{pesoPorDia:.2f} kg by day")
print(f"R$ {gastoPorDia:.2f} by day")

File: problemas/ad-hoc/1217/1217.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let pesoPorDia = 0.0, gastoPorDia = 0.0;

let N = parseInt(lines.shift().trim());

for(let i = 1; i <= N; ++i){
    let V = parseFloat(lines.shift().trim());
    let frutas = lines.shift().trim().split(' ').length;

    console.log(`day ${i}: ${frutas} kg`);

    pesoPorDia += frutas;
    gastoPorDia += V;
}

pesoPorDia /= N;
gastoPorDia /= N;

console.log(`${pesoPorDia.toFixed(2)} kg by day`);
console.log(`R$ ${gastoPorDia.toFixed(2)} by day`);
